saas/models/saas_db.py

```diff
@@ -41,3 +41,18 @@ def drop_db(self):
     def get_url(self):
         self.ensure_one()
         return self.operator_id.get_db_url(self)
+
+    @api.multi
+    @job
+    def auth_built_post_init(self):
+        self.ensure_one()
+        # fixme: not sure about method of getting master_url
+        master_url = self.env['ir.config_parameter'].get_param('web.base.url')
+        db = sql_db.db_connect(self.name)
+        registry(self.name).check_signaling()
+        with api.Environment.manage(), db.cursor() as cr:
```

Comment:
It works only for local database. You have to check operator type. Move it to 'saas.operator' model (you may leave a wrapper here similar to get_url)